modules/sprite.js

```diff
@@ -40,19 +44,52 @@ displaySystem.registerModule({
         function addSprite(config) {
             let sprite = document.createElement('div');
             sprite.className = 'sprite';
-            sprite.innerHTML = config.html || '';
+            var imageServer = "http://".concat(window.location.hostname).concat(":1395/");
+            var imgSrc = imageServer.concat(config.alias);
+
+
+            var img = document.createElement('img');

```

Comment:
The idea of sprites is that they can take generic html. This is far too specific around images